src/main/java/uk/gov/ons/ctp/response/collection/exercise/repository/CollectionExerciseRepository.java

```diff
@@ -60,4 +60,22 @@
           nativeQuery = true)
   String getActiveActionPlanId(@Param("p_exercisefk") Integer exercisefk,
       @Param("p_sampleunittypefk") String sampleunittypefk, @Param("p_surveyuuid") UUID surveyuuid);
+
+  /**
+   * Query repository for list of collection exercises associated with a certain party ID.
+   *
+   * @param partyid for which to return Collection Exercises.
+   * @return collection exercises for party ID.
+   */
+  @Query(value = "select ce.id, ce.exercisepk, ce.name, ce.scheduledstartdatetime, ce.scheduledexecutiondatetime, "
```

Comment:
Can you shorten query to SELECT ce FROM collectionexercise.collectionexercise ce ...

Reply:
Tried this but it didn't work, probably because of the joins